packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,272 @@
 # @fluentui/react-persona Spec
 
+Convergence epic issue: #24213
+
 ## Background
 
-_Description and use cases of this component_
+A Persona is a visual representation that showcases an Avatar, Icon, Image, or Badge.
+
+Persona is used in PeoplePicker, Team's left rail and menus, chiclets, and Card.
 
 ## Prior Art
 
-_Include background research done for this component_
+- OpenUI research: [Avatar/Persona](https://open-ui.org/components/avatar.research)
+- v7/v8: [link](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+  - > Note: v8's Persona is a combination of v9's Persona and Avatar (v8's Persona has a sub-component PersonaCoin that would count as Avatar, but is not exposed.). In v9 we've taken the approach of having a single component Avatar and another component Persona.
+- v0: Does not have equivalent component.
+
+### Coin
+
+_Avatar vs PersonaCoin_: As mentioned in Prior Art, v8's persona is not quite the same as v9's. v8's Persona is only able to showcase PersonaCoin while v9 showcases Avatar (including Avatar + PresenceBadge, Avatar + image, and Avatar + Icon).
```

Comment:
This is the first place you reference PersonaCoin.  It would be helpful to anyone reading that isn't familiar with the differences between Avatar and PersonaCoin to have a visual to show their differences (or description of the visual differences).